src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private TaskManager manager = new TaskManager();
+
+    public void exit() {
```

Comment:
You may want to consider changing the name of this method to make it more descriptive, as it could be confused for System.exit()